### PR TITLE
Java cc37

### PR DESCRIPTION
**Pull request by wafman**
- [x] Top-level README “Table of Contents” is updated
 - [x] Feature tasks for this challenge are completed
 - [x] Unit tests written and passing
     - [x] “Happy Path” - Expected outcome
     - [x] Expected failure
     - [x] Edge Case (if applicable/obvious)
 - [x] README for this challenge is complete
     - [x] Summary, Description, Approach & Efficiency, Solution
     - [x] Link to code
     - [x] Picture of whiteboard

### CODE REVIEW

File: code401Challenges/src/main/java/code401Challenges/GetEdge/GetEdge.java [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+import code401Challenges.Graph.Node;
+
+import java.util.HashSet;
+
+public class GetEdge<T> {
+
+    boolean output = false;
+    int cost = 0;
+
+    public GetEdge(){}
```
Comment (evanslaton) [inline code401Challenges/src/main/java/code401Challenges/GetEdge/GetEdge.java:14]: You don't need a no-args constructor unless you're working with Spring and a database

File: code401Challenges/src/main/java/code401Challenges/GetEdge/GetEdge.java [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+        for(Node node : h){
+            if(node.label.equals(arr[0])){
+                current = node;
+            }
+        }
+
+
+        if(current == null){
+            GetEdge answer = new GetEdge(false, 0);
```
Comment (evanslaton) [inline code401Challenges/src/main/java/code401Challenges/GetEdge/GetEdge.java:36]: Since you have `new GetEdge(false, 0)` in your code twice, why not save it to a variable near the top of the function and return it (changing it if it should be `true`)?
